### PR TITLE
 Drop phase 2 for kube-scheduler.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@sttts @sjenning

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -196,7 +198,7 @@ podman run \
 	--network=host \
 	--entrypoint=/bootkube \
 	"{{.BootkubeImage}}" \
-	start --asset-dir=/assets --required-pods openshift-kube-apiserver/openshift-kube-apiserver,kube-system/kube-scheduler,openshift-kube-controller-manager/openshift-kube-controller-manager,openshift-cluster-version/cluster-version-operator
```
Comment (sttts) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: the scheduler should not be dropped here, just the name must be adapted to phase 3.

**Comment by aveshagarwal** [comment]
@sttts updated. PTAL.

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 2]
```diff
@@ -88,7 +88,9 @@ then
 		--manifest-image=${OPENSHIFT_HYPERKUBE_IMAGE} \
 		--asset-input-dir=/assets/tls \
 		--asset-output-dir=/assets/kube-scheduler-bootstrap \
-		--config-output-file=/assets/kube-scheduler-bootstrap/config
+		--config-output-file=/assets/kube-scheduler-bootstrap/config \
+		--config-override-files=/assets/bootkube-config-overrides/kube-scheduler-config-overrides.yaml \
```
Comment (sttts) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: where is this coming from? the bootstrap override file is applied internally in the operator render command.

**Comment by sjenning** [comment]
wfm.  `kube-scheduler` is gone from `kube-system` and scheduler in `openshift-kube-scheduler` installs and runs as expected.

**Comment by aveshagarwal** [comment]
all tests passed, lgtm/approval?

**Comment by abhinavdahiya** [comment]
i think all the override files from installer should be dropped to discovery based. but we can do that later on..

**Pull request merged**